code/_helpers/global_lists_vr.dm

```diff
@@ -435,6 +436,22 @@ var/global/list/contamination_flavors_musky = list("drenched",
 				"sticky",
 				"tainted")
 
+var/global/list/contamination_flavors_slimy = list("slimy",
+				"sloppy",
+				"drippy",
+				"glistening",
+				"dripping",
+				"gunky",
+				"slimed",
+				"mucky",
+				"viscous",
+				"dank",
+				"glutinous",
+				"syrupy",
+				"slippery",
+				"gelatinous",
+				"thick")
```

Comment:
These are good, but if I might offer a point of critique, I think 'thick' might look a bit funny

'thick headband'
'thick Soanso's ID card', etc.

Reply:
thicc